snabbdom.js

```diff
@@ -30,14 +31,18 @@ function createKeyToOldIdx(children, beginIdx, endIdx) {
 
 function createRmCb(childElm, listeners) {
   return function() {
-    if (--listeners === 0) childElm.parentElement.removeChild(childElm);
+    if (--listeners === 0) {
+      var parent = api.parentNode(childElm);
+      api.removeChild(parent, childElm);
+    }
   };
 }
 
 var hooks = ['create', 'update', 'remove', 'destroy', 'pre', 'post'];
 
-function init(modules) {
+function init(modules, domApi) {
```

Comment:
I don't mind. You can change any names you think would be more suitable. For example, I changed parentElement to parentNode but maybe it should stay parentElement.
```
function parentNode(node){
  return node.parentElement;
}
```